src/instructlab/defaults.py

```diff
@@ -78,6 +78,7 @@ class _InstructlabDefaults:
     MERLINITE_GGUF_REPO = "instructlab/merlinite-7b-lab-GGUF"
     MISTRAL_GGUF_REPO = "TheBloke/Mistral-7B-Instruct-v0.2-GGUF"
     GRANITE_GGUF_MODEL_NAME = "granite-7b-lab-Q4_K_M.gguf"
+    GRANITE_EMBEDDINGS_MODEL_NAME = "ibm-granite/granite-embedding-125m-english"
```

Comment:
should be `EMBEDDING `?  without `S`?

Reply:
I think the S is redundant 
@anastasds ?